src/WebJobs.Script/Binding/ServiceBusScriptBindingProvider.cs

```diff
@@ -61,6 +60,23 @@ public override void Initialize()
                 }
             }
 
+            EventProcessorOptions eventProcessorOptions = EventProcessorOptions.DefaultOptions;
+            eventProcessorOptions.MaxBatchSize = 1000;
+            configSection = (JObject)Metadata.GetValue("eventHub", StringComparison.OrdinalIgnoreCase);
+            if (configSection != null)
+            {
+                if (configSection.TryGetValue("maxBatchSize", StringComparison.OrdinalIgnoreCase, out value))
+                {
+                    eventProcessorOptions.MaxBatchSize = (int)value;
+                }
+
+                if (configSection.TryGetValue("prefetchCount", StringComparison.OrdinalIgnoreCase, out value))
+                {
+                    eventProcessorOptions.PrefetchCount = (int)value;
+                }
+            }
+            _eventHubConfiguration = new EventHubConfiguration(eventProcessorOptions);
```

Comment:
I changed the ordering during host init that Initialize is called. Now I call Initialize early, before we attempt to create any bindings. This makes more sense, and was actually required for these updates, since we need to read the config info to get the EventProcessorOptions before creating any bindings (since the EventHubConfiguration is passed into the bindings).